tinker/faculty_bios/__init__.py

```diff
@@ -85,6 +85,13 @@ def delete_confirm(self):
         return render_template('faculty-bios/delete-confirm.html')
 
     def new(self):
+        if 'Tinker Faculty Bios - Admin' in session['groups'] or 'Administrators' in session['groups'] \
```

Comment:
This should probably be moved to forms.py

Reply:
Fixed